files/scripts/graphdb.sh

```diff
@@ -98,4 +105,87 @@ function createRepositoryFromFile {
   fi
 }
 
+function interpolate {
+  local input=""
+  while IFS='' read -r line || [ -n "$line" ]; do
+    input="$input$line"$'\n'
+  done
+
+  vars=$(echo "${input}" | grep -o "\${[^}]*}" | sed 's/[${}]//g' | uniq)
+
+  local output="$input"
+  for var in $vars; do
+    value=${!var:-''}
+    if [ -n "$value" ]; then
+      output=${output//\$\{$var\}/$value}
+    fi
+  done
+  echo "${output}"
+}
+
+function cloudBackup {
+  BACKUP_TIMESTAMP="$(date +'%Y-%m-%d_%H-%M-%S')"
+  BACKUP_NAME="graphdb-backup-${BACKUP_TIMESTAMP}.tar"
+  export BACKUP_NAME BACKUP_TIMESTAMP
```

Comment:
why are these exported?

Reply:
Leftovers from when I tried using `envsubst` (which required vars to be exported)

The current solution should work without exporting them, so I'll remove it.

Reply:
btw, `envsubst` should work like this (no export required):
```shell
FOO="bar" \
BAZ="biz" \
envsubst ...
```